nucypher/blockchain/eth/sol/source/contracts/proxy/Dispatcher.sol

```diff
@@ -81,9 +85,9 @@ contract Dispatcher is Upgradeable {
     * @dev Fallback function send all requests to the target contract.
     * If contract not exists then result will be unpredictable (see DELEGATECALL)
     **/
-    function () public payable {
-        assert(target != 0x0);
-        bool callSuccess = target.delegatecall(msg.data);
+    function () external payable {
```

Comment:
Here too - why public to external?

Reply:
the same, now fallback function only external